projects/fsastorefrontlib/src/cms-components/my-account/quote/quote-comparison/quote-comparison.component.html

```diff
@@ -38,8 +38,7 @@ <h2 class="heading-headline p-3 mt-3 mb-5 text-center">
               </div>
               <div *ngFor="let cart of quotes?.carts" class="col">
                 {{
-                  ((cart?.deliveryOrderGroups)[0]?.entries)[0]?.product
-                    ?.cartDisplayName
+                  ((cart?.deliveryOrderGroups)[0]?.entries)[0]?.product?.name
```

Comment:
deliveryOrdreGroups[0] is not needed, just go with 
                `  cart?.entries[0]?.product?.name`

Reply:
deliveryOrdreGroups[0] removed